src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+.page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.page__body {
+  margin: 0;
+}
+
+h1,
+h2 {
+  margin: 0;
+}
+
+span {
+  display: block;
+}
```

Comment:
Be careful with global styles for HTML tags. It's better to use class selectors to avoid unexpected styles in other parts of your project. You can create a class like `.block-display` and apply it where you need.